docs/features/init.md

```diff
@@ -42,6 +42,12 @@ type = 'workspace' # type specifying if the project is a workspace or standalone
 projects_root_path = 'projects' # path to the root folder containing all sub-projects in the workspace
 ```
 
+#### VSCode optimizations
+
+AlgoKit has a set of minor optimizations for VSCode users that are useful to be aware of:
+
+- Templates created with the `--workspace` flag automatically include a VSCode code-workspace file. New projects added to an AlgoKit workspace are also integrated into an existing VSCode workspace.
```

Comment:
Are these two options the default?

Reply:
@robdmoore the --ide one has been there since the start, i noticed it wasn't documented so figured ill note it down explicitly. As for the first one its a new behaviour that this pr is adding based on feedback from @iskysun96 and discussion over it with @neilcampbell